pkg/benchmarks/run

```diff
@@ -29,7 +30,7 @@ function padRight(text, length) {
 }
 
 function center(text, length) {
-  let padLength = (length - text.length) / 2;
+  const padLength = (length - text.length) / 2;
 
   return dup(" ", Math.floor(padLength))
     + text
```

Comment:
i realize this is old code that didn't come from you and i'm just seeing it because of the `let` -> `const` switch, but, also this is weird code and i don't like it.  there's all sorts of bizarre failure cases, like if length is shorter than the source string, that are just not failing by chance

```typescript
function center(text, length) {

  const emptyLength = Math.max(length - text.length, 0),
        leftPad     = Math.floor(emptyLength / 2),
        rightPad    = emptyLength - leftPad;

  return (' '.repeat(leftPad)) + (text) + (' '.repeat(rightPad));

}
```